content/md/what-is/what-is-an-asset.md

```diff
@@ -0,0 +1,51 @@
+---
+id: what-is-an-asset
+themes: first-steps, akeneo-concepts, manage-your-assets
+title: What is an **asset**?
+popular: false
+ee-only: true
+related: assets-transformation, upload-assets, work-with-assets
+---
+
+# Definition of an asset
+
+You will use an asset whenever you want to enrich your products with files. These files can be, for example, pictures or user manuals of your products.
+
+::: ee
+This feature is only available in the Entreprise Edition.
+:::
+
+One asset is composed of several files depending on if you need a different version of this asset for each of your locales or channels.
+
+![Asset](../img/Settings_What-is-an-asset.svg)
+
+An asset has a unique code which is mandatory and optional properties :
+- A set of files: one or several reference files and their variations per channel,
+- A description,
+- An end date of use,
+- Some tags (to qualify the asset and find it quickly).
+
+As stated above, an asset can be localizable. In this case, you will have as many reference files than activated locales. It is very useful when you need, for example, one PDF file for your manual in English and another one in French.
+
+To finish, you can define different transformations of these assets according to your channel needs, i.e. a picture with different sizes depending on the channels. To learn more about these transformations, you can go [here](assets-transformation.html).
+
+# Why should I use assets?
+
+There are 2 major benefits when you use the assets to manage the files linked to your products:
+- you can easily manage them from one single place,
+- you can easily link them to your products.
```

Comment:
Can we add: 
- Images are automatically transformed according to your needs and will then match your export needs ?